src/taco-remote-lib/ios/ios.ts

```diff
@@ -218,6 +218,15 @@ class IOSAgent implements ITargetPlatform {
                 res.status(200).json(buildInfo.localize(req, resources));
             }
         });
+
+        ideviceinstaller.on("error", function (err: any): void {
+            res.status(500);
+            if (err.code === "ENOENT") {
+                res.send(resources.getStringForLanguage(req, "IdeviceinstallerNotFound"));
```

Comment:
Which casing are you referring to? The string ID here is new, and if you prefer a different casing then I can change it.